src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RefreshConstants.cs

```diff
@@ -21,5 +21,9 @@ internal class RefreshConstants
         // Backoff during refresh failures
         public static readonly TimeSpan DefaultMinBackoff = TimeSpan.FromSeconds(30);
         public static readonly TimeSpan DefaultMaxBackoff = TimeSpan.FromMinutes(10);
+
+        // Timeouts to retry requests to primary config stores after HTTP status codes 503 or 429.
+        public static readonly TimeSpan DefaultMinRetryAfter = TimeSpan.FromSeconds(30);
```

Comment:
Why can't we use the the two above? (DefaultMinBackoff, DefaultMaxBackoff)

Reply:
If it's because we don't want them to be coupled then perhaps these don't belong in RefreshConstants.